websites/D/DiscordJS Guide/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "DiscordJS Guide",
+    "author": {
+      "name": "OGMatrix",
+      "id": "657290880947978241"
+    },
+    "category": "other",
+    "logo": "https://i.imgur.com/Zt4i2l3.png",
+    "thumbnail": "https://i.imgur.com/ojmsp7f.png",
+    "version": "1.0.1",
+    "color": "#20f856",
+    "tags": [
+      "ogmatrix",
```

Comment:
``ogmatrix``, delete this tag. This tag is not affiliated with this website, or is it?